packages/cyberstorm/src/components/Markdown/Markdown.module.css

```diff
@@ -0,0 +1,145 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  color: var(--color-text--default);
+  font: var(--font-body);
+  font-weight: 500;
+  line-height: 1.6rem;
+}
+
```

Comment:
Forgot to mention this last time around. Let's clear the padding on the first item of the document by adding this after the first rule: `.root > :first-child { padding-top:0; }`